### PR TITLE
docs: update info for the Operator local installation

### PR DESCRIPTION
**Pull request by vbelouso**
## Description

Update conversation cache settings to use only Redis

## Type of change

- [ ] Refactor
- [ ] New feature
- [ ] Bug fix
- [ ] CVE fix
- [ ] Optimization
- [x] Documentation Update
- [ ] Configuration Update
- [ ] Bump-up dependent library

## Related Tickets & Documents

- Related Issue #
- Closes #

## Checklist before requesting a review

- [x] I have performed a self-review of my code.
- [x] PR has passed all pre-merge test jobs.
- [ ] If it is a core feature, I have added thorough tests.

## Testing
- Please provide detailed steps to perform tests related to this code change.
- How were the fix/results from this change verified? Please provide relevant screenshots or results.

### CODE REVIEW

**Comment by raptorsun** [comment]
Thank you for such a quick fix  @vbelouso  =) 
Could you please update the example CR file  `config/samples/ols_v1alpha1_olsconfig.yaml`, too?

**Comment by vbelouso** [comment]
> Thank you for such a quick fix @vbelouso =) Could you please update the example CR file `config/samples/ols_v1alpha1_olsconfig.yaml`, too?

I removed the sample from the docs because users try to use it directly without modifying it to suit their needs.
Such as `a default working config`, which in our case is not true because of the LLM provider's secrets.

File: README.md [hunk 1]
```diff
@@ -1,87 +1,55 @@
...
-A Kubernetes operator for managing OpenShift Lightspeed service.
+A Kubernetes operator for managing [Red Hat OpenShift Lightspeed](https://github.com/openshift/lightspeed-service).
 
 ## Getting Started
 
-You'll need a OpenShift cluster to run against.
+You'll need an OpenShift cluster to run against.
 
-**Note:** Your controller will automatically use the current context in your kubeconfig file (i.e. whatever cluster `oc cluster-info` shows).
+> [!IMPORTANT]
```
Comment (raptorsun) [inline README.md:9]: shall we use `**IMPORTANT**` here to make it bold?
This does not render special format in my markdown preview ^^" 
image

Comment (vbelouso) [reply README.md:9]: It's [GH-specific syntax](https://docs.github.com/en/get-started/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax#alerts)

https://github.com/openshift/lightspeed-operator/blob/a5f041141a351457655d2b5e7db1a75a0b2d80fa/README.md

**Pull request merged**